api/types.go

```diff
@@ -604,6 +605,7 @@ func DefaultOptions() Options {
 		MirostatEta:      0.1,
 		PenalizeNewline:  true,
 		Seed:             -1,
+		ControlStrength:  []string{},
```

Comment:
Although these values should be floats, we save them as strings to adding avoid extra type casting logic to the options parser